.github/workflows/test.yml

```diff
@@ -40,13 +40,6 @@ jobs:
         uses: actions/checkout@v2
       - name: Run tests 
         run: GO_VERSION=${{ matrix.version }} make docker-test
-      - name: Codecov upload
```

Comment:
and also just curious why we have to remove these too?

Reply:
I didn’t want to go through the setup for codecov right now, but maybe we’ll do that in the future (I think you have to setup an account, etc)